src/main/java/core/basesyntax/MainApp.java

```diff
@@ -2,4 +2,17 @@
 
 public class MainApp {
 
+  Machine excavator = new Excavator();
+  Machine bulldozer = new Bulldozer();
+  Machine truck = new Truck();
```

Comment:
The instance variables 'excavator', 'bulldozer', and 'truck' should be private to adhere to the principle of encapsulation. This ensures that the internal representation of the object is hidden from the outside, only allowing access through methods.